packages/core/upload/admin/src/hooks/__mocks__/useConfig.js

```diff
@@ -0,0 +1,13 @@
+export const useConfig = jest.fn().mockReturnValue({
+  config: {
+    isLoading: false,
+    isError: false,
+    data: {
+      data: {
+        pageSize: 10,
+      },
+    },
```

Comment:
Do you think it would be possible the remove the nested `data` attributes here? I understand where it is coming from, but I don't find it very intuitive to use ...